docs/standard/collections/commonly-used-collection-types.md

```diff
@@ -20,7 +20,7 @@ Collections are based on the [ICollection](https://docs.microsoft.com/dotnet/cor
+Collections can vary, depending on how the elements are stored, how they are sorted, how searches are performed, and how comparisons are made. The [Queue](https://docs.microsoft.com/dotnet/core/api/System.Collections.Queue) class and the [Queue&lt;T&gt;](https://docs.microsoft.com/dotnet/core/api/System.Collections.Generic.Queue-1) generic class provide first-in-first-out lists, while the [Stack](https://docs.microsoft.com/dotnet/core/api/System.Collections.Stack) class and the [Stack&lt;T&gt;](https://docs.microsoft.com/dotnet/core/api/System.Collections.Generic.Stack-1) generic class provide last-in-first-out lists. The [SortedList](https://docs.microsoft.com/dotnet/core/api/System.Collections.SortedList) class and the [SortedList&lt;TKey, TValue&gt;](https://docs.microsoft.com/dotnet/core/api/System.Collections.Generic.SortedList-2) generic class provide sorted versions of the [Hashtable](https://docs.microsoft.com/dotnet/core/api/System.Collections.Hashtable) class and the [Dictionary&lt;TKey, TValue&gt;](https://docs.microsoft.com/dotnet/core/api/System.Collections.Generic.Dictionary-2) generic class. The elements of a `Hashtable` or a `Dictionary<TKey, TValue>` are accessible only by the key of the element, but the elements of a `SortedList` or a [KeyedCollection&lt;TKey, TItem&gt;](https://docs.microsoft.com/dotnet/core/api/System.Collections.ObjectModel.KeyedCollection-2) are accessible either by the key or by the index of the element. The indexes in all collections are zero-based, except [Array](https://docs.microsoft.com/dotnet/core/api/System.Array), which allows arrays that are not zero-based.
```

Comment:
Good change.

I think we should use the code blocks where we have links, for consistency and simplicity. For example:

...and the [`Queue<T>`](https://...)

What do you think?